packages/py-api/py_api/controllers/hackathon_teams_controller.py

```diff
@@ -0,0 +1,124 @@
+import json
+import random
+import string
+from typing import Literal
+
+from bson.json_util import dumps
+from bson.objectid import ObjectId
+from fastapi.responses import JSONResponse
+from py_api.database.initialize import t_col
+from py_api.models import UpdateTeam
+from py_api.utilities.parsers import filter_none_values
+
+
+class TeamsController:
+    @classmethod
+    def create_team(cls, team_name: str, user_id: str) -> Literal[True] | Literal[False]:
```

Comment:
What if we want to create a new team in the admin panel so that we can move some of the members there? We won't be able to reuse this controller as is. 

Consider adding another endpoint for moving members from one team to another so that we can manage this from the frontend.

Reply:
Ok